ui/src/app/edge/live/common/grid/modal/constants.spec.ts

```diff
@@ -55,4 +56,21 @@ export const LINE_HORIZONTAL: OeFormlyViewTester.Field = {
 export const LINE_INFO_PHASES_DE: OeFormlyViewTester.Field = {
   type: "info-line",
   name: "Die Summe der einzelnen Phasen kann aus technischen Gründen geringfügig von der Gesamtsumme abweichen."
-};
\ No newline at end of file
+};
+
+export const ONLY_NAME_LINE = (name: string): OeFormlyViewTester.Field => ({
```

Comment:
Is `NAME_LINE` not sufficient?

Reply:
I thought Name-Line is not to explicit, it doesnt tell me, that there is only a name in this line

Reply:
It is clear by the fact, that the object type only takes a 'name'. I would change it.

Reply:
ok